assessments/CLI_CIRCLE/templates/studentAssessments.jsont

```diff
@@ -0,0 +1,127 @@
+        "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Integer",
+        "result": "{{ subject_score }}"
+      },
+      {
+        "assessmentReportingMethodDescriptor": "uri://www.cli.org/circle/AssessmentReportingMethodDescriptor#Subject",
+        "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
+        "result": "{{ subject_descriptor }}"
+      }
+    ]
+        
+    "studentObjectiveAssessments": [
+      {%- for n in range(1,8) -%}
+      
+        {%- set objective_assessment_name = self._TemplateReference__context.resolve('oa_category_'~n~'_name') -%}
+        {%- set objective_assessment_descriptor = self._TemplateReference__context.resolve('oa_category_'~n~'_descriptor') -%}
+        {%- set objective_assessment_date = self._TemplateReference__context.resolve('oa_category_'~n~'_date') -%}
+        {%- set objective_assessment_score = self._TemplateReference__context.resolve('oa_category_'~n~'_score') -%}
+        {%- set objective_assessment_performance = self._TemplateReference__context.resolve('oa_category_'~n~'_performance') -%}
+
+        {%- if objective_assessment_score is not none and objective_assessment_score | length -%}
+          {%- if not loop.first -%},{% endif %}
+        {
+          "administrationDate": "{{ objective_assessment_date }}",
+          "objectiveAssessmentReference": {
+            "assessmentIdentifier": "cli_circle",
+            "identificationCode": "{{ subject_name }}"
+            "namespace": "uri://www.cli.org/circle/Assessment"
+          },          
+          "performanceLevels": [
+            {
+              "assessmentReportingMethodDescriptor": "uri://www.cli.org/circle/AssessmentReportingMethodDescriptor#Cut Point",
+              "resultDatatypeTypeDescriptor":"uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
+              "performanceLevelDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#{{ objective_assessment_performance }}"
+            }
+          ],
+          "scoreResults": [
+            {
+              "assessmentReportingMethodDescriptor": "uri://www.cli.org/circle/AssessmentReportingMethodDescriptor#Score",
+              "resultDatatypeTypeDescriptor":"uri://ed-fi.org/ResultDatatypeTypeDescriptor#Integer",
+              "result": "{{ objective_assessment_score }}"
+            },
+            {
+              "assessmentReportingMethodDescriptor": "uri://www.cli.org/circle/AssessmentReportingMethodDescriptor#Subject",
+              "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
+              "result": "{{ objective_assessment_descriptor }}"
+            },
+            {
+              "assessmentReportingMethodDescriptor": "uri://www.cli.org/circle/AssessmentReportingMethodDescriptor#Wave",
+              "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
+              "result": "{{ wave }}"
```

Comment:
is wave an attribute of obj_assess result, overall assess result, or both?

Reply:
It is only really relevant in the case of sets, [per this conversation](https://github.com/edanalytics/earthmover_edfi_bundles/pull/32#discussion_r1442305943), and right now the results for each set is outputted under ObjectiveAssessments.